docs/reference/api/docker_remote_api_v1.23.md

```diff
@@ -374,8 +374,8 @@ Json Parameters:
     -   **Binds** – A list of volume bindings for this container. Each volume binding is a string in one of these forms:
            + `host_path:container_path` to bind-mount a host path into the container
            + `host_path:container_path:ro` to make the bind-mount read-only inside the container.
-           + `volume_name:container_path` to bind-mount a volume managed by a volume plugin into the container.
-           + `volume_name:container_path:ro` to make the bind mount read-only inside the container.
+           + `volume_name:container_path[:nocopy]` to bind-mount a volume managed by a volume plugin into the container. Volume contents are initialized with data from `container-dest`, provided that the volume is empty. Such an initialization can be avoided by using the `nocopy` option.
+           + `volume_name:container_path:ro[,nocopy]` to make the bind mount read-only inside the container. Volume contents are initialized with data from `container-dest`, provided that the volume is empty. Such an initialization can be avoided by using the `nocopy` option.
```

Comment:
Given that there's more options possible than just `:ro` and `no copy` (such as `Z` and `z`), I wonder if we should use `[options]` here, and list the available options (mentioning that it's a comma-separated list). For example this is how it's described in the `docker run` CLI reference;

```
 -v, --volume value                Bind mount a volume (default []). The format
                                    is `[host-src:]container-dest[:<options>]`.
                                    The comma-delimited `options` are [rw|ro],
                                    [z|Z], [[r]shared|[r]slave|[r]private], and
                                    [nocopy]. The 'host-src' is an absolute path
                                    or a name value.
```

That format is not ideal, but a bullet-list could work (and probably more readable) also have a look at https://github.com/docker/docker/pull/26791, which reorganizes the run-reference